README.md

```diff
@@ -60,7 +60,7 @@ func main() {
 
 	// Create a new Mux and set the handler
 	mux := http.NewServeMux()
-	mux.HandleFunc("/events", server.HTTPHandler)
+	mux.HandleFunc("/events", server)
```

Comment:
This should be ` server.ServeHTTP`, as `Mux` expects a regular http handler for it's routes.